src/quo/components/animated/pressable.cljs

```diff
@@ -69,26 +70,29 @@
          :or                 {border-radius 0
                               type          "primary"}}
         (bean/bean props)
-        long-press-ref       (react/create-ref)
-        state                (animated/use-value (:undetermined gesture-handler/states))
-        active               (animated/eq state (:began gesture-handler/states))
-        gesture-handler      (animated/use-gesture {:state state})
-        animation            (react/use-memo
-                              (fn []
-                                (animated/with-timing-transition active
-                                  {:duration (animated/cond* active time-in time-out)
-                                   :easing   (:ease-in animated/easings)}))
-                              [])
+        long-press-ref (react/create-ref)
+        state (animated/use-value (:undetermined gesture-handler/states))
+        active (animated/eq state (:began gesture-handler/states))
+        gesture-handler (animated/use-gesture {:state state})
+        animation (react/use-memo
+                   (fn []
+                     (animated/with-timing-transition active
+                                                      {:duration (animated/cond* active time-in time-out)
+                                                       :easing   (:ease-in animated/easings)}))
+                   [])
```

Comment:
that's a n interesting case. Although original code doesn't is aligned to a (almost) random position, after change it is not aligned at all.

Reply:
@rasom, @yqrashawn:

I'm curious too, but it seems to be related to the heuristic to only align let bindings up to a certain threshold. If that's the case I see two choices:

1. Always right-align let bindings and map keys, even if one of the keys or the binding name is very long in comparison to the others. This would end the discussion, but there'd be some edge cases. I'm more inclined to this option as it's the most predictable, because it'll generate less diff in future PRs.
2. Second option, increase the threshold and see if it helps.

Reply:
Third option, but most people don't like it: simply never right align map keys and let bindings. I used to write a lot of Clojure code like this, but yeah, I know most people would hate it.

Reply:
This one is not affected by the `:max-variance`, which is wired, need more time look into this